src/js/components/ServiceInfo.js

```diff
@@ -93,6 +93,14 @@ class ServiceInfo extends React.Component {
     let runningTasksCount = tasksSummary.tasksRunning;
     let instancesCount = service.getInstancesCount();
     let runningTasksSubHeader = StringUtil.pluralize('Task', runningTasksCount);
+    let overCapacity = '';
+    let isDeploying = serviceStatus === 'Deploying';
+
+    if (tasksSummary.tasksOverCapacity !== 0) {
```

Comment:
Let's change this condition to `tasksOverCapacity > 0`, to not accidentally render the message if  `tasksOverCapacity` is undefined or null.